stronk/utils/auth.py

```diff
@@ -21,6 +21,8 @@ def verify_token():
             # Verify the ID token while checking if the token is revoked by
             # passing check_revoked=True.
             decoded_token = auth.verify_id_token(id_token, check_revoked=True)
+            # Set the Firebase Token in the request context
+            g.firebase_token = id_token
             # Token is valid and not revoked.
             return decoded_token['uid']
```

Comment:
I should've caught this earlier, but we shouldn't return here because it's an early return. Removing this will allow the actual request to reach the graphql endpoint

Reply:
@13ABEL oops good catch!